### PR TITLE
Feature/calculator component

### PR DESCRIPTION
**Pull request by MacCrazyman**
# Add basic calculator structure 💻 📱 ⌨️ 

This branch implements React components to create a calculator based on the wireframe of the project.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+import CalcButton from './button';
+
+function Calculator() {
```
Comment (rloterh) [inline src/components/calculator.js]: - [x] Your _Calculator_ component _(Calculator.js)_ is not classed based. Kindly change this component to a class-based component.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +0,0 @@
-import logo from './logo.svg';
```
Comment (rloterh) [inline src/App.js:1]: - [x] App.js is not present in your application. This milestone requires App.js and it should be a class-based component. Kindly endeavor to improve upon this aspect

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Leonardo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,10 @@
...
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```
Comment (SajjadAhmad14) [inline src/App.js]: Please make `App` component a class based component.

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
```
Comment (SajjadAhmad14) [inline src/App.js:2]: [ OPTIONAL ] Please remove all boilerplate that got created by the `Create React App` (texts, images, and styles).

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Leonardo, 
 ### Thank you for working on changes suggested previously :ok_hand:

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by D4nielJ — APPROVED** [review]
## Hello @MacCrazyman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**